levels/fullstack.tex

```diff
@@ -0,0 +1,251 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=800mm, paperheight=500mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills able.
+\newcolumntype{C}{p{18em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Fullstack Engineering
+\end{huge}
+\end{center}
+
+% Alternating row colors.
+\rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{tabular}{ | C | C | C | C | C | C | C | C |}
+  \hline
+  \rowcolor{blue!60!black!10}
+    Level/Skill &
+
+    \cellcolor{blue!25}
+    \textbf{Product Output} &
+    \textbf{Data structures/Performance} &
+    \textbf{Security} &
+    \textbf{Communication/Writing} &
+    \textbf{Networking} &
+    \textbf{Fullstack Engineering} &
+    \textbf{Tooling}
+    \\
+  \hline
+    1&
+
+    \cellcolor{blue!25}
+    Collaborates with product through designs, mockups, and demos of product
+    features before implementation.
+    &
+
+    Demonstrates good understanding of basic data structures like hash tables,
+    linked lists, and trees. Can reason about algorithm complexity. Applies relevant
+    data structures in day to day activity.
+
+    \bigbreak
+
+    Can implement a production quality software - it might be not the most
+    efficient or secure, but correct.
+    &
+
+    Understands basics of web security (can configure TLS on a standalone web
+    server like nginx).
+    &
+
+    Reports progress on a regular basis as required by the team's operational
+    requirements. Actively solicits feedback. Participates on interview panels.
+    &
+
+    Understands and reasons about browser networking concepts. Understands
+    common networking issues and can troubleshoot issues using the developer
+    console.
+    &
+
+    Understands and can write HTML.
+    &
+
+    Understands and can use build tools (like npm and yarn), Docker, and
+    Makefiles.
+    \\ [13em]
+  \hline
+    2&
+
+    \cellcolor{blue!25}
+    Can write high quality user and product focused documentation.
+    &
+
+    &
+
+    Understands how to utilize browser security features (like headers and
+    cookie attributes) to secure web applications.
+    &
+
+    Provides constructive review on peers' code and design. Helps new team
+    members during their first weeks.
+    &
+
+    Has more granular understanding of the networking design, for example can
+    reason about using gRPC vs HTTP/JSON and websockets and their networking
+    and scalability trade-offs. Understands mTLS.
```

Comment:
Compared to other requirements for lvl 2, this particular requirement seems to be much more advanced.

The Networking column seems to be mostly a copy of the column from the systems eng levels. Would it be reasonable to bump Networking lvl 2 and 3 to lvl 3 and 4 for fullstack levels?